### PR TITLE
doc: fix typo, indention etc.

### PR DESCRIPTION
**Pull request by xiexingguo**
Signed-off-by: xiexingguo <xie.xingguo@zte.com.cn>

### CODE REVIEW

File: doc/rados/operations/crush-map.rst [hunk 1]
```diff
@@ -100,14 +99,14 @@ the hook off in the configuration::
 Custom location hooks
 ---------------------
 
-A customize location hook can be used in place of the generic hook for OSD
-daemon placement in the hierarchy.  (On startup, each OSD ensure its position is
+A customized location hook can be used in place of the generic hook for OSD
+daemon placement in the hierarchy(On startup, each OSD ensure its position is
```
Comment (theanalyst) [inline doc/rados/operations/crush-map.rst]: nit, we should probably mention each OSD _ensures_ its position is correct

Comment (xiexingguo) [inline doc/rados/operations/crush-map.rst]: thanks a lot. I will fix it as soon as I am back to work:-)

File: doc/rados/operations/crush-map.rst [hunk 2]
```diff
@@ -100,14 +99,14 @@ the hook off in the configuration::
 Custom location hooks
 ---------------------
 
-A customize location hook can be used in place of the generic hook for OSD
-daemon placement in the hierarchy.  (On startup, each OSD ensure its position is
+A customized location hook can be used in place of the generic hook for OSD
+daemon placement in the hierarchy(On startup, each OSD ensures its position is
```
Comment (tchaikov) [inline doc/rados/operations/crush-map.rst]: any reason to remove the period and the following spaces before the parenthesis?

File: doc/rados/operations/crush-map.rst [hunk 3]
```diff
@@ -100,14 +99,14 @@ the hook off in the configuration::
...
 correct.)::
 
   osd crush location hook = /path/to/script
 
 This hook is passed several arguments (below) and should output a single line
-to stdout with the CRUSH location description.::
+to stdout with the CRUSH location description::
```
Comment (tchaikov) [inline doc/rados/operations/crush-map.rst]: why reason for removing the period?

**Comment by xiexingguo** [comment]
@tchaikov Thank you for your attention. Dropped those changes since they are irrelevant:-)

**Comment by theanalyst** [comment]
LGTM

**Comment by xiexingguo** [comment]
@tchaikov @theanalyst  Thank you for your careful review :+1:

**Pull request merged**